extensions/omniv21/validation/ediFileDeclaration.json

```diff
@@ -42,7 +42,8 @@
                             "index": { "type": "integer", "minimum": 1 },
                             "component_index": { "type": "integer", "minimum": 1 },
                             "empty_if_missing": { "type": "boolean","$comment": "deprecated, use 'default'" },
-                            "default": { "type": "string" }
+                            "default": { "type": "string" },
+                            "_comment": { "$ref": "#/definitions/value_comment" }
```

Comment:
Actually, just realized, can you at least add some `_comment` into existing EDI samples in `extensions/omniv21/samples/edi`?

Reply:
Ah yes good call!